crates/interpreter/src/instruction_result.rs

```diff
@@ -91,6 +91,50 @@ impl From<Halt> for InstructionResult {
     }
 }
 
+#[macro_export]
+macro_rules! return_ok {
+    () => {
+        InstructionResult::Continue
+            | InstructionResult::Stop
+            | InstructionResult::Return
+            | InstructionResult::SelfDestruct
+    };
+}
+
+#[macro_export]
+macro_rules! return_revert {
+    () => {
+        InstructionResult::Revert | InstructionResult::CallTooDeep | InstructionResult::OutOfFund
+    };
+}
+
```

Comment:
```suggestion

#[macro_export]
```

Reply:
I have considered exporting the macro to the whole crate, but decided that the macro should be kept local/private until/unless the need to access it from the outside appears in the future.

Wdyt?

Reply:
It is a utility macro and could be useful. It is strange to half export `return_ok` and `return_revert` and not export the third one.

Reply:
Included the suggested export into #961